docs/source/index.md

```diff
@@ -0,0 +1,138 @@
+# Xpublish
+
+```{warning}
+Don't mind the mess, our docs are currently being restructured.
+```
+
+**Useful links:** [Installation](getting-started/installation) | [Source Repository](https://github.com/xpublish-community/xpublish/) | [Issue Tracker](https://github.com/xpublish-community/xpublish/issues) | [Q&A Support](https://github.com/xpublish-community/xpublish/discussions/categories/q-a?discussions_q=category%3AQ%26A+) | [Slack Channel](./ecosystem/index.md#slack)
+
+Xpublish is:
+
+::::{grid} 3
+
+:::{grid-item-card} A quick way to serve a single Xarray dataset
+:link: getting-started/tutorial/introduction
+
+Add a REST API to your dataset with `ds.rest.serve()`!
```

Comment:
this feels duplicative

Reply:
The whole card, or 'Add a REST API to your dataset with `ds.rest.serve()`?

How about something like 'Get started with `ds.rest.serve()` to explore serving data with Xpublish'?